app/js/main.js

```diff
@@ -254,19 +250,42 @@ function startup() {
     // Listen for leaving frame rate input
     inputChangeFR.addEventListener("blur", function() {
         inputChangeFR.value = frameRate;
-        if(inputChangeFR.value > 60 || inputChangeFR.value < 1 || NaN || inputChangeFR.length > 2) {
+        if (inputChangeFR.value > 60 || inputChangeFR.value < 1 || NaN || inputChangeFR.length > 2) {
             inputChangeFR.value = 15;
         }
     });
 
     // Toggle capture and playback windows
     btnLiveView.addEventListener("click", function() {
-        if (winMode === "capture") {
-            winMode = "playback";
-        } else if (winMode === "playback") {
-            winMode = "capture";
+        // Switch from frame preview back to live view
+        if (document.querySelector(".frame-reel-preview.selected")) {
```

Comment:
Would it be better for `.frame-reel-preview.selected` to be a new variable or to keep it like this?